MAUI/ImageEditor/save.md

```diff
@@ -215,4 +215,30 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+### Save picker opening event
+
+The `SavePickerOpening` event occurs while the save picker is opening on the save icon clicked in toolbar.
```

Comment:
The `SavePickerOpening` event occurs while the save picker opens on the save icon clicked in the toolbar.

Reply:
updated.